src/components/Tooltip/Tooltip.tsx

```diff
@@ -154,8 +155,13 @@ export default class Tooltip extends PureComponent<TooltipProps> {
     return (
       <div
         style={style}
-        className={classnames(
-          `monday-style-tooltip monday-style-tooltip-${theme} padding-size-${paddingSize}`,
+        className={cx(
+          styles.tooltip,
+          "monday-style-tooltip",
+          getStyle(styles, camelCase(theme)),
+          `monday-style-tooltip-${theme}`,
+          getStyle(styles, camelCase("padding-size-" + paddingSize)),
```

Comment:
I think the specific  undefined issue which we saw "paddingSizeUndefined" maybe not related to the other undefined issues in the snapshots and maybe happen because of a bug here.... can we check in the storybook that the classnames are valid?